41hulk/ewawe-pay
# Payment Features

#### What does this PR do?
Give the user all Payment features

#### Description of Task to be completed?
- Implemented the dashboard button
- Implemented the data entry of User screen
- Implemented the Confirmation Screen
- Implemented a selection option of payment methods with appropriate modal for data entry

#### How should this be manually tested?
`git fetch origin ft-payrent`

Then checkout to this branch
`git checkout ft-payrent`

Get all dependencies if there are any
`flutter pub get`

Make sure you have an emulator or a device connected to your computer then
Run the application with this command
`flutter run`

#### Screenshots (if appropriate)
